client/my-sites/pages/page/index.js

```diff
@@ -351,6 +352,13 @@ class Page extends Component {
 		);
 	}
 
+	getQRCodeItem() {
+		if ( ! config.isEnabled( 'post-list/qr-code-link' ) ) {
+			return null;
+		}
+		return <PostActionsEllipsisMenuQRCode globalId={ this.props.page.global_ID } key="bug" />;
```

Comment:
Is the 'bug' key still good here now that we have our own icon?

Reply:
Done!